src/main/java/me/dinowernli/grpc/polyglot/config/CommandLineArgs.java

```diff
@@ -84,6 +84,13 @@
       usage="If true, then the message specification for the method is rendered")
   private String withMessageArg;
 
+  //TODO: Move to a "list_services"-specific flag container
+  @Option(
+          name = "--json_output",
```

Comment:
Can we have this be a --list_output_format and have "json" be one of the valid flag values?